src/instructlab/model/train.py

```diff
@@ -462,7 +462,7 @@ def get_files(directory: str, pattern: str) -> list[str]:
             shutil.rmtree(final_results_dir)
         final_results_dir.mkdir()
 
-        gguf_models_dir = Path("./models")
+        gguf_models_dir = Path(DEFAULTS.MODELS_DIR)
```

Comment:
(For other reviewers) This is also sent as https://github.com/instructlab/instructlab/pull/1765 Probably better to merge 1765 before this PR.

Reply:
So why do we put the trained model into cache directory? Shouldn't it go to checkpoints? (As proposed in the other PR.)

Reply:
This was written before that PR merged, I'll update this.

Reply:
OK. This PR should be updated to include the checkpoints dir patch (assuming you are trying to collect everything relevant here? Not sure what the plan is.)

Reply:
This test is written under the assumption that this is the change

Reply:
Is it correct though? Weren't the trained models supposed to be stored in persistent store (which is not where `MODELS_DIR` point to - under `.cache`)?

```
    @property
    def MODELS_DIR(self) -> str:
        return path.join(self._cache_home, STORAGE_DIR_NAMES.MODELS)
```

Reply:
Thank you for catching this @booxter, I tried to accept the current changes in the rebase but I guess I accidentally accepted the incoming ones instead. Updating this to the correct path.

Reply:
Fixed.